### PR TITLE
fix(mf1.5): add resolve alias for mf runtime

### PR DESCRIPTION
**Pull request by ahabhgk**
## Require Documentation?

- [ ] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

File: packages/rspack/src/container/ModuleFederationPlugin.ts [hunk 1]
```diff
@@ -147,9 +171,7 @@ function getDefaultEntryRuntime(
 		runtimePluginVars.push(`${runtimePluginVar}()`);
 	}
 	const content = [
-		`import __module_federation_implementation__ from ${JSON.stringify(
-			implementationPath
-		)}`,
+		`import __module_federation_bundler_runtime__ from "@module-federation/runtime-tools/webpack-bundler-runtime"`,
```
Comment (h-a-n-a) [inline packages/rspack/src/container/ModuleFederationPlugin.ts:174]: Is this intended to be hardcoded?

Comment (ahabhgk) [reply packages/rspack/src/container/ModuleFederationPlugin.ts]: No, forgot change

**Comment by h-a-n-a** [comment]
Additionally, I took a look at the `package.json` file. It turns out module federation related dependencies are not added. https://github.com/web-infra-dev/rspack/blob/c24cd5506895547acb274b122449a62abc078308/packages/rspack/package.json#L84

It may cause pnpm resolution issues. Would you please add?

**Comment by ahabhgk** [comment]
I added it to dependencies, should I also add it to peer?

**Comment by h-a-n-a** [comment]
> I added it to dependencies, should I also add it to peer?

Sorry, I misunderstand your intention. If module-federation dependencies are intended to be provided by '@rspack/core' then it's not necessary.

**Comment by ahabhgk** [comment]
it's in the @module-federation/runtime-tools, and resolved from @module-federation/runtime-tools

**Comment by h-a-n-a** [comment]
I see. Then, it's good to go ;-)

**Review by h-a-n-a — APPROVED** [review]

**Pull request merged**